python/run-tests.sh

```diff
@@ -56,10 +56,13 @@ set -e
 # Horrible hack for spark 1.4: we manually remove some log lines to stay below the 4MB log limit on Travis.
 # To remove when we ditch spark 1.4.
```

Comment:
Unfortunately, I see a lot of logs being generated for the spark 1.6 run. The spark 2 travis run does not print logs, as expected from the log4j file.